codes/data/multi_country/get_adm_info.py

```diff
@@ -215,6 +224,7 @@ def main():
         }
     }
     for iso3 in isos:
+        print(f"Downloading and processing {iso3} geographical and population data...")
```

Comment:
check the f prefix

Reply:
good call! will fix

Reply:
oh whoops nevermind thats right. it's an fstring in python...which is the same as `"blahblah{a}".format(a=someothervar)`